x-pack/plugins/cases/public/components/markdown_editor/editor.tsx

```diff
@@ -34,6 +34,7 @@ interface MarkdownEditorProps {
   parsingPlugins?: PluggableList;
   processingPlugins?: PluggableList;
   uiPlugins?: EuiMarkdownEditorUiPlugin[] | undefined;
```

Comment:
I know it is unrelated to this PR but the `uiPlugins` in combination with the `disabledUiPlugins` confused me. Do you mind if we remove the `uiPlugins` prop? I don't think is being used anywhere.

Reply:
I did notice it was not used and wanted to remove it too. I'll remove it in this PR.

Reply:
Thanks!